presto-docs/src/main/sphinx/release/release-0.267.rst

```diff
@@ -0,0 +1,30 @@
+=============
+Release 0.267
+=============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Add ``@CodegenScalarFunction`` to add scalar functions using codegen.
```

Comment:
What's the user-facing impact of this one?

Reply:
@pranjalssh ,  Can you clarify what is the user facing impact for this ? .

Reply:
This makes it simpler to add user defined scalar functions which use codegen. Also provides a way to hook them to plugins.

Reply:
@pranjalssh Removing this as per our offline discussion